aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,59 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use crate::ServiceDefinition;
+
+pub fn restart(services: &[&ServiceDefinition]) {
+    // stop all services
+    for ServiceDefinition {
+        service,
+        sockets: _,
+    } in services
+    {
+        print!("Stopping {}...", service);
+        let result = Command::new("systemctl")
+            .args(&["stop", service])
+            .output()
+            .unwrap();
```

Comment:
Bubble up the errors using `.with_context(|| format!("could not stop {}", ...))` instead of `.unwrap()` (`use anyhow::Context;` . See the rest of the aziotctl code for examples.)

Same for the `status.success()` check.

Reply:
This is in `aziotctl-common`, which doesn't currently depend on `anyhow`.

I'm all for adding it as a dependency, since `anyhow` is pretty slick, and plays nicely with `std::error::Error`. Alternatively, we could just have a simple module-specific error enum.

In either case, `unwrap` 🙅